GooglePay/src/main/java/com/braintreepayments/api/GooglePayActivityResultContract.java

```diff
@@ -33,7 +33,7 @@ public GooglePayResult parseResult(int resultCode, @Nullable Intent intent) {
                 return new GooglePayResult(PaymentData.getFromIntent(intent), null);
             }
         } else if (resultCode == RESULT_CANCELED) {
-            return new GooglePayResult(null, new UserCanceledException("User canceled Google Pay."));
+            return new GooglePayResult(null, new UserCanceledException("User canceled Google Pay.", true));
```

Comment:
We set this as true because there isn't a way to multi-task out of the Google Pay sheet. This activity result is set to canceled when the user manually clicks the X out of the pay sheet.